### PR TITLE
Migrate elf council proposals

### PR DESCRIPTION
**Pull request by hnguyen1179**

### CODE REVIEW

File: packages/council-proposals/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      "type": "pwa-chrome",
+      "request": "launch",
+      "url": "http://localhost:3000",
+      "webRoot": "${workspaceFolder}/frontend/src",
+      "sourceMapPathOverrides": {
+        "webpack:///frontend/src/*": "${webRoot}/*"
+      }
+    }
+  ]
+}
```
Comment (DannyDelott) [inline packages/council-proposals/.vscode/launch.json:15]: nit: Remove this file

File: packages/council-proposals/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
```
Comment (DannyDelott) [inline packages/council-proposals/.vscode/settings.json:1]: nit: Remove this file

File: apps/elf-council-frontend/package.json [hunk 3]
```diff
@@ -4,7 +4,6 @@
     "@babel/types": "^7.17.0",
     "@blueprintjs/popover2": "^1.0.0",
     "@elementfi/base": "^1.0.0",
-    "@elementfi/elf-council-proposals": "^1.0.2",
```
Comment (DannyDelott) [inline apps/elf-council-frontend/package.json:7]: We'll need to add the `@elementfi/council-proposals` here and run `yarn` to rebuild yarn.lock

File: packages/council-proposals/package.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@elementfi/council-proposals",
+  "version": "1.0.2",
+  "description": "",
+  "dependencies": {
+    "@elementfi/elf-council-tokenlist": "^1.0.2",
+    "@elementfi/elf-council-typechain": "^1.0.2",
```
Comment (DannyDelott) [inline packages/council-proposals/package.json:7]: hmm, looks like we'll need to migrate these projects over as well:

https://github.com/element-fi/elf-council-typechain
https://github.com/element-fi/elf-council-tokenlist

They're currently published to npm so everything still works here, but let's do this in a follow up PR

File: packages/council-proposals/package.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    "typescript": "^4.3.5"
+  },
+  "scripts": {
+    "build-testnet": "bash scripts/build-testnet.sh && npm run _build-index && npm run _copy-files",
+    "build-goerli": "bash scripts/build-goerli.sh && npm run _build-index && npm run _copy-files",
+    "build-mainnet": "bash scripts/build-mainnet.sh",
+    "build": "npm run build-goerli && npm run build-mainnet && npm run build-testnet && npm run _build-index && npm run _copy-files",
+    "_build-index": "bash scripts/build-index.sh",
+    "_copy-files": "bash scripts/copy-dist-files.sh",
+    "_copy-testnet-addresses": "cp ../elf-council-testnet/src/addresses/testnet.addresses.json src/addresses/testnet.addresses.json",
```
Comment (DannyDelott) [inline packages/council-proposals/package.json:53]: elf-council-testnet is now council-testnet in the monorepo, `../apps/council-testnet/src/addresses/testnet.addresses.json`

File: apps/elf-council-frontend/package.json [hunk 6]
```diff
@@ -4,6 +4,7 @@
     "@babel/types": "^7.17.0",
     "@blueprintjs/popover2": "^1.0.0",
     "@elementfi/base": "^1.0.0",
+    "@elementfi/council-proposals": "1.0.2",
```
Comment (hnguyen1179) [inline apps/elf-council-frontend/package.json:7]: (No change to yarn.lock)

**Review by DannyDelott — APPROVED** [review]

**Pull request merged**